tests/kernels/test_attention.py

```diff
@@ -92,6 +92,7 @@ def ref_single_query_cached_kv_attention(
         output[i].copy_(out, non_blocking=True)
 
 
+# TODO(woosuk): Add tests for USE_ALIBI=True.
```

Comment:
Is this TODO outdated?

Reply:
Oh, this was my mistake. I moved this to `test_multi_query_kv_attention`, which does not have a test for ALiBi yet.